test/specs/views/Items/Item-test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {Item} from 'stardust';
+
+describe('Item', () => {
+  it('has children', () => {
+    let renderedItem = render(
+      <Item description='discription'>
```

Comment:
added this test to confirm we receive a consoled error regarding description and children for an Item component.